daemon/src/model/cfd.rs

```diff
@@ -226,6 +227,8 @@ pub enum CfdState {
         common: CfdStateCommon,
         dlc: Dlc,
         attestation: Option<Attestation>,
+        // TODO: Unsure about whether we need to keep this if we're pending commit
+        collaborative_close: Option<(Transaction, SystemTime)>,
```

Comment:
:wrench: I think we should remove this field. If we ever transition back to `Open` from this state collaborative close is then possible again. And we should never be able to transition from `Open` to `PendingCommit` with this field set to `Some(_)`, so no information should be lost.

Reply:
👍  removed it!